LanguageExt.CodeGen/LanguageExt.CodeGen.csproj

```diff
@@ -22,26 +17,15 @@
     <PackageLicenseUrl>https://github.com/louthy/language-ext/blob/master/LICENSE.md</PackageLicenseUrl>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <DocumentationFile></DocumentationFile>
-    <OutputType>library</OutputType>
     <AssemblyVersion>3.0.0.0</AssemblyVersion>
     <FileVersion>3.0.0.0</FileVersion>
     <NoWarn>1701;1702;1705;IDE1006;CS1591;CS1573;CS1712;CS1570;CS1711;CS1572;CS1587</NoWarn>
-
+    <IncludeBuildOutput>false</IncludeBuildOutput>
```

Comment:
In stable release, I've updated the Metapackage.Sdk to include that by default so you can remove it here :)

Reply:
Sure!